queue-exp/pifo_srpt.py

```diff
@@ -36,15 +36,15 @@
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 """
 
-from pifo_lib import Packet, Runner, Pifo
-from pifo_lib import SchedulingAlgorithm
-from pifo_lib import FlowTracker
+from sched_lib import Packet, Runner, Pifo, SchedulingAlgorithm
+from sched_lib import FlowTracker
 
 
 class Srpt(SchedulingAlgorithm):
     """Shortest Remaining Processing Time"""
 
-    def __init__(self):
+    def __init__(self, name=None):
```

Comment:
Yes, you are right. I did add it after. However, I decided to keep it with the HPFQ commit because that is what drove that change. When you run the HPFQ example, it was hard to see which reference referred to the left and right WFQ in the root, so I added the names to the algorithms to distinguish them. It would have been easier to follow in a separate commit. I will try to keep that in mind in the future when I make changes like this.

Reply:
Totally get why you needed it! However, when reviewing, such "unrelated" changes make it harder to pick out the important bits, so it's better to split them into a separate commit and just point out that they will be used by the subsequent commit... So yeah, please split them in the future :)